submissions/LofeWilter/HTML-CSS-Popup/style.css

```diff
@@ -0,0 +1,134 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    width: 100%;
+    height: 100%;
+}
+
+.container { 
+    margin: 0 auto;
+    width: 40%;
+    display: flex;
+    flex-direction: column;
+}
+
+.nav {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+}
+
+.nav_item {
+    padding: 5px 10px;
+    color: #66646F;
+    font-size: 16px;
+    align-self: center;
+}
+
+.nav_item img {
+    width: 24px;
+    height: 24px;
+    cursor: pointer;
+}
+
+.wrapper {
+    display: none;
+    align-self: flex-end;
+    width: 252px;
+    height: 335px;
+    overflow-y: auto;
+}
+
+.services {
+    flex-direction: row;
+    flex-wrap: wrap;
+    display: flex;
+    margin: 2px;
+}
+
+figure {
+    padding: 5px;
+    margin: 0 2px;
+}
+
+figure img {
+    width: 50px;
+    height: 35px;
+    margin: 5px;
+    cursor: pointer;
+} 
+
+figcaption {
+    text-align: center;
+    color: #66646F;
+    font-size: 12px;
+}
+
+.more {
+    width: 100%;
+    text-align: center;
+    background-color: #D2D2D2;
+    color: #66646F;
+    opacity: 0.8;
+    height: 30px;
+    line-height: 30px;
+}
+
+.more div {
+    width: 100%;
+}
+
+#moreservices:checked ~ .wrapper {
+    display: block;
+}
+
+#moreservices {
```

Comment:
the same as in the previous comment.
It's better to use these styles for hiding `input`:
``` CSS
.visually-hidden {
	position: absolute !important;
	clip: rect(1px 1px 1px 1px); /* IE6, IE7 */
	clip: rect(1px, 1px, 1px, 1px);
	padding:0 !important;
	border:0 !important;
	height: 1px !important; 
	width: 1px !important; 
	overflow: hidden;
}
```
More info [here](https://htmlacademy.ru/blog/boost/frontend/short-12)